tests/parallel/test_processor.py

```
@@ -499,5 +500,88 @@ def test_shutdown_is_safe(self) -> None:
+
+        out = processor._handle_timeouts(
+            pending=pending,
+            future_paths=future_paths,
+            future_started=future_started,
+            future_queued_at=future_queued_at,
+            timeout=0.1,
+            poll_interval=0.05,
+            finalize_result=lambda r: r,
+        )
+        self.assertIsNotNone(out)
+        should_abort, needs_shutdown, results = out  # type: ignore[misc]
```

Ruff (0.15.17)

[warning] Unpacked variable `needs_shutdown` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)